util/shell_exec.go

```diff
@@ -69,3 +64,49 @@ func PassthruCommand(cmd *exec.Cmd) (exitCode int) {
 
 	return
 }
+
+// RunCommand executes the provided command, it also can sspecify if the output should be forced to print to the console
+func (x Executor) Execute(forceOutput bool) error {
+	x.cmd.Stderr = os.Stderr
+	if Logger().IsVerbose || forceOutput {
+		x.cmd.Stdout = os.Stdout
+	} else {
+		x.cmd.Stdout = ioutil.Discard
+	}
+
+	x.Log("Executing")
+	color.Set(color.FgCyan)
+	err := x.Run()
+	color.Unset()
+	return err
+}
+
+// Run runs a command via exec.Run() without modification or output of the underlying command.
+func (x Executor) Run() error {
+	return x.cmd.Run()
```

Comment:
Do we want to log all commands that are run? Or only commands that force/stream?

Reply:
Probably all commands, but I wanted to lay out the initial API for it more flexibly so the refactoring can go either way.